profile/profile.go

```diff
@@ -143,6 +152,7 @@ func (p Profile) Encode() (*config.ProfilePod, error) {
 	for _, maddr := range p.NetworkAddrs {
 		addrs = append(addrs, maddr.String())
 	}
+	// TODO(aqru): see if we should also show pubkey/keyID here
```

Comment:
@b5 This is still TODO before I merge

Reply:
@dustmop any opinion on this?

Reply:
I think this is a no? We have the profileID, and can use that to look up the key{public,id} from the keystore. No reason to store it twice, better to keep it normalized.